app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,101 +1,59 @@
 import React, { useState, useEffect } from "react";
 
-import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
-import EmptyState from "components/Common/EmptyState";
-
-import DeleteAlert from "./DeleteAlert";
+import { NOTES } from "./constants";
+import NotesList from "./NotesList";
+import NotesMenu from "./NotesMenu";
 import NewNotePane from "./Pane/Create";
-import Table from "./Table";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
-  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
-  const [notes, setNotes] = useState([]);
+  const [notes, setNotes] = useState([{}]);
+  const [showMenu, setShowMenu] = useState(false);
 
   useEffect(() => {
-    fetchNotes();
+    setNotes(NOTES);
+    setLoading(false);
   }, []);
 
-  const fetchNotes = async () => {
-    try {
-      setLoading(true);
-      const { data } = await notesApi.fetch();
-      setNotes(data.notes);
-    } catch (error) {
-      logger.error(error);
-    } finally {
-      setLoading(false);
-    }
-  };
-
   if (loading) {
     return <PageLoader />;
   }
 
   return (
     <Container>
+      <NotesMenu showMenu={showMenu} />
       <Header
-        title="Notes"
+        title="All Notes"
         actionBlock={
           <Button
             icon="ri-add-line"
             label="Add New Note"
             onClick={() => setShowNewNotePane(true)}
           />
         }
+        menuBarToggle={() => {
+          setShowMenu(!showMenu);
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

Reply:
This is resolved
I have set it as 

`menuBarToggle={() => setShowMenu(show => !show)}`